typings/mongoengine/fields.pyi

```diff
@@ -756,38 +755,36 @@ class EmbeddedDocumentField(Generic[_ST, _GT], BaseField):
     @overload
     def __new__(
         cls,
-        document_type: Type[_Doc_Type],
+        document_type: Type[_T],
         required: Literal[False] = ...,
-        default: None = ...,
         help_text: str = ...,
-    ) -> EmbeddedDocumentField[Optional[_Doc_Type], Optional[_Doc_Type]]: ...
+    ) -> EmbeddedDocumentField[Optional[_T], Optional[_T]]: ...
     @overload
     def __new__(
         cls,
-        document_type: Type[_Doc_Type],
+        document_type: Type[_T],
         required: Literal[False] = ...,
         *,
-        default: Union[_Doc_Type, Callable[[], _Doc_Type]],
+        default: Union[_T, Callable[[], _T]],
         help_text: str = ...,
-    ) -> EmbeddedDocumentField[Optional[_Doc_Type], _Doc_Type]: ...
+    ) -> EmbeddedDocumentField[Optional[_T], _T]: ...
     @overload
     def __new__(
         cls,
-        document_type: Type[_Doc_Type],
+        document_type: Type[_T],
         *,
         required: Literal[True],
-        default: None = ...,
```

Comment:
deleting the `default` arg appeases pylance. Pylance doesn't like the possibility of `_T` being nullable